sdlib/d/gc/region.d

```diff
@@ -186,14 +223,9 @@ private:
 			return null;
 		}
 
-		static assert(ExtentSize / Region.sizeof == 2,
-		              "Unexpected Region size!");
-
-		auto r0 = Region.fromSlot(slot, 0);
-		auto r1 = Region.fromSlot(slot, 1);
+		static assert(Region.sizeof <= ExtentSize, "Unexpected Region size!");
```

Comment:
Isn't there a way to make it fit? It seems wasteful to multiply by two the amount of space taken by the metadata to store one size.